server/src/internalClusterTest/java/org/opensearch/cluster/NoMasterNodeIT.java

```diff
@@ -90,7 +90,7 @@ protected int numberOfReplicas() {
     public void testNoMasterActions() throws Exception {
         Settings settings = Settings.builder()
             .put(AutoCreateIndex.AUTO_CREATE_INDEX_SETTING.getKey(), true)
-            .put(NoMasterBlockService.NO_MASTER_BLOCK_SETTING.getKey(), "all")
+            .put(NoMasterBlockService.NO_CLUSTER_MANAGER_BLOCK_SETTING.getKey(), "all")
```

Comment:
Is there something preventing us from updating NoMasterBlockService as well?

Reply:
Sorry just saw your response to kartg with the same question.

Reply:
Hi @mch2 thanks for your review! Actually there is nothing blocked. The main reason not to change the class or method name in this PR is to control the size of the code changes in a single PR. Another reason is there are too many class or method that have been published to maven needs renaming, and I didn't target in version 2.0.0 for them.